concepts/05 UI Components/Tooltip/02 Getting Started with Tooltip/10 Customize Tooltip.md

```diff
@@ -0,0 +1,192 @@
+
+    <!-- tab: app.component.html -->
+    <div class="tooltips">
+        <dx-tooltip
+            contentTemplate="Like"
+        ></dx-tooltip>
+        <dx-tooltip
+            contentTemplate="trashTooltip"
+        >
+            <div *dxTemplate="let model of 'trashTooltip'">
+                <div class="red-tooltip">Delete</div>
+            </div>
+        </dx-tooltip>
+        <dx-tooltip
+            contentTemplate="infoTooltip"
+        >
+            <div *dxTemplate="let model of 'infoTooltip'">
+                <div class="blue-tooltip">Info</div>
+            </div>
+        </dx-tooltip>
+    </div>
+
+##### Vue
+
+To customize Tooltip content, specify [contentTemplate](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate). If you assign a string to this property, Tooltip displays plain text in its container:
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="tooltips">
+            <DxTooltip
+                content-template="Like"
+            />
+            <DxTooltip
+                content-template="trashTooltip"
+            >
+                <template #trashTooltip>
+                    <div class="red-tooltip">Delete</div>
+                </template>
+            </DxTooltip>
+            <DxTooltip
+                content-template="infoTooltip"
+            >
+                <template #infoTooltip>
+                    <div class="blue-tooltip">Info</div>
+                </template>
+            </DxTooltip>
+        </div>
+    </template>
+
+##### React
+
+To customize Tooltip content, specify [contentRender](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentRender) or [contentComponent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentComponent):
+
+    <!-- tab: App.tsx -->
+    function likeRender(): JSX.Element {
+        return <div>Like</div>;
+    }
+
+    function trashRender(): JSX.Element {
+        return (
+            <div className='red-tooltip'>Delete</div>
+        );
+    }
+
+    function infoRender(): JSX.Element {
+        return (
+            <div className='blue-tooltip'>Info</div>
+        );
+    }
+
+    function App(): JSX.Element {
+        return (
+            <div className="tooltips">
+                <Tooltip
+                    contentRender={likeRender}
+                />
+                <Tooltip
+                    contentRender={trashRender}
+                />
+                <Tooltip
+                    contentRender={infoRender}
+                />
+            </div>
+        )
+    }
+
+---
+
+You can customize the appearance of Tooltip visual elements with CSS. This example implements the following styles to change the colors of `#trash-tooltip` and `#info-tooltip`:
```

Comment:
As I see, not all platforms use these IDs - "trash-tooltip" and "info-tooltip". For example, React sample doesn't have those names. Maybe we should just mention associated classes. And maybe we need to mention that we specifies CSS class names in code snippets above. 

```suggestion
You can use CSS to customize Tooltip appearance. You may have noticed that code snippets above assigned CSS classes to all tooltips. This example defines those styles (`.red-tooltip` and `.blue-tooltip`):
```